source/Stargate-API-Skeleton/StargateApplication.class.st

```diff
@@ -48,12 +65,24 @@ StargateApplication class >> logsDirectory [
 	^ ( FileLocator workingDirectory / 'logs' ) ensureCreateDirectory
 ]
 
+{ #category : #private }
+StargateApplication class >> projectName [
+
+	^ self subclassResponsibility
```

Comment:
unnecessary return here

Reply:
It's true, but I use it as an indicator that the implementors are expected to explicitly return something